frontend/lib/src/components/core/Block/styled-components.ts

```diff
@@ -136,8 +136,7 @@ export const StyledVerticalBlock = styled.div<StyledVerticalBlockProps>(
   ({ width, theme }) => ({
     width,
     position: "relative", // Required for the automatic width computation.
-
-    display: "flex",
+    display: width === 0 ? "none" : "flex",
```

Comment:
I think we might not even need it. But I think in any case if the `width` of a vertical container is 0 we should better not display the container.